frontend/src/static/js/api/client.ts

```diff
@@ -18,6 +18,28 @@ import createClient, {HeadersOptions, type FetchOptions} from 'openapi-fetch';
 import {type components, type paths} from 'webstatus.dev-backend';
 import {createAPIError} from './errors.js';
 
+// import type {PathsWithMethod} from 'openapi-typescript-helpers';
+
+// Local copy of PathsWithMethod from openapi-typescript-helpers,
+// since eslint complains that the type is not required, but it is.
+type HttpMethod =
+  | 'get'
+  | 'put'
+  | 'post'
+  | 'delete'
+  | 'options'
+  | 'head'
+  | 'patch'
+  | 'trace';
+
+type PathsWithMethod<Paths extends {}, PathnameMethod extends HttpMethod> = {
+  [Pathname in keyof Paths]: Paths[Pathname] extends {
+    [K in PathnameMethod]: unknown;
+  }
+    ? Pathname
+    : never;
+}[keyof Paths];
+
```

Comment:
Inside the frontend directory, run: `npm install -D openapi-typescript-helpers`. This will fix the `n/no-extraneous-import` error. But it will create a new error: `n/no-unpublished-import`. To fix that, add it to the package to this existing list: https://github.com/GoogleChrome/webstatus.dev/blob/8e603ebd03939f013d359fee09abf8058fd10919/frontend/eslint.config.mjs#L78

Afterwards, you should be able to do the following and the lint command should succeed.
```suggestion
import {type PathsWithMethod} from 'openapi-typescript-helpers';

```

Reply:
That worked, thanks.  My openapi-typescript-helpers appeared to be up-to-date.